spider/unsupervised_learning/grasta/grasta.py

```diff
@@ -391,38 +388,38 @@ def calculate_mla_step(grastaSTATUS, grastaOPTIONS, admm_OPTS, gamma, w, sG):
                 if step >= np.pi / 3:
                     step = np.pi / 3
 
-            bShrUpd = 0
-
-            if mu <= MIN_MU:
-                if level > 1:
-                    bShrUpd = 1
-                    level = level - 1
-
-                mu = DEFAULT_MU_LOW
-
-            elif mu > MAX_MU:
-                if level < MAX_LEVEL:
-                    bShrUpd = 1
-                    level = level + 1
-                    mu = DEFAULT_MU_HIGH
-                else:
-                    mu = MAX_MU
-
-            if bShrUpd:
-                if level >= 0 and level < 4:
-                    MAX_ITER = grastaOPTIONS.admm_min_itr
-                elif level >= 4 and level < 7:
-                    MAX_ITER = min(MIN_ITER * 2, ITER_MAX)
-                elif level >= 7 and level < 10:
-                    MAX_ITER = min(MIN_ITER * 4, ITER_MAX)
-                elif level >= 10 and level < 14:
-                    MAX_ITER = min(MIN_ITER * 8, ITER_MAX)
+                bShrUpd = 0
+
+                if mu <= MIN_MU:
+                    if level > 1:
+                        bShrUpd = 1
+                        level = level - 1
+
+                    mu = DEFAULT_MU_LOW
+
+                elif mu > MAX_MU:
+                    if level < MAX_LEVEL:
+                        bShrUpd = 1
+                        level = level + 1
+                        mu = DEFAULT_MU_HIGH
+                    else:
+                        mu = MAX_MU
+
+                if bShrUpd:
+                    if level >= 0 and level < 4:
+                        MAX_ITER = grastaOPTIONS.admm_min_itr
+                    elif level >= 4 and level < 7:
+                        MAX_ITER = min(MIN_ITER * 2, ITER_MAX)
+                    elif level >= 7 and level < 10:
+                        MAX_ITER = min(MIN_ITER * 4, ITER_MAX)
+                    elif level >= 10 and level < 14:
+                        MAX_ITER = min(MIN_ITER * 8, ITER_MAX)
+                    else:
+                        MAX_ITER = ITER_MAX
                 else:
                     MAX_ITER = ITER_MAX
-            else:
-                MAX_ITER = ITER_MAX
```

Comment:
@kgilman Unindent this. As is, this undoes a change I made in 4630bbede0d5b75709edab8c969376c5ec220baf.

Reply:
Actually, let me try this myself first. Sorry for the confusion.